cfme/tests/automate/generic_objects/buttons/test_generic_class_custom_button.py

```diff
@@ -332,9 +350,8 @@ def test_generic_object_button_delete_flash(button_without_group):
     )
 
 
-@pytest.mark.manual
 @pytest.mark.meta(blockers=[BZ(1753281), BZ(1753388)], coverage=[1753281, 1753388])
```

Comment:
The `coverage` list here should be updated to `automates`, as the meta marker processes them differently for BZ reports.